pages/common/dune.md

```diff
@@ -0,0 +1,20 @@
+# dune
+
+> A build system for [OCaml](https://ocaml.org) programs.
+> More information: <https://dune.build>.
+
+- Build all targets:
+
+`dune build`
+
+- Clean up the workspace:
+
+`dune clean`
+
+- Run tests:
+
+`dune runtest`
+
+- Load compiled libraries in the [utop](https://github.com/ocaml-community/utop) REPL:
```

Comment:
Could you explain what this does.

Reply:
I reworded the passage. Let me know if it's still not clear.